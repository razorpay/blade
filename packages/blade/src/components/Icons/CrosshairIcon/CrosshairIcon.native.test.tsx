import CrosshairIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.native';

describe('<CrosshairIcon />', () => {
  it('should render CrosshairIcon', () => {
    const renderTree = renderWithTheme(
      <CrosshairIcon color="feedback.icon.neutral.intense" size="large" />,
    ).toJSON();
    expect(renderTree).toMatchSnapshot();
  });
});
