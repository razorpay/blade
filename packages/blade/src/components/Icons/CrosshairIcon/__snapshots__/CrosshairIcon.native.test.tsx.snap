// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CrosshairIcon /> should render CrosshairIcon 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <RNSVGSvgView
    accessibilityElementsHidden={true}
    align="xMidYMid"
    bbHeight="20px"
    bbWidth="20px"
    data-blade-component="icon"
    fill="none"
    focusable={false}
    height="20px"
    importantForAccessibility="no-hide-descendants"
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        [
          {},
        ],
        {
          "flex": 0,
          "height": 20,
          "width": 20,
        },
      ]
    }
    vbHeight={24}
    vbWidth={24}
    width="20px"
  >
    <RNSVGGroup
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        clipRule={0}
        d="M1 12C1 5.92487 5.92487 1 12 1C18.0751 1 23 5.92487 23 12C23 18.0751 18.0751 23 12 23C5.92487 23 1 18.0751 1 12ZM13 20.9451V18C13 17.4477 12.5523 17 12 17C11.4477 17 11 17.4477 11 18V20.9451C6.82838 20.4839 3.51608 17.1716 3.05493 13H6C6.55228 13 7 12.5523 7 12C7 11.4477 6.55228 11 6 11H3.05493C3.51608 6.82838 6.82838 3.51608 11 3.05493V6C11 6.55228 11.4477 7 12 7C12.5523 7 13 6.55228 13 6V3.05493C17.1716 3.51608 20.4839 6.82838 20.9451 11H18C17.4477 11 17 11.4477 17 12C17 12.5523 17.4477 13 18 13H20.9451C20.4839 17.1716 17.1716 20.4839 13 20.9451Z"
        fill={
          {
            "payload": 4280563015,
            "type": 0,
          }
        }
        fillRule={0}
        propList={
          [
            "fill",
            "fillRule",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
